pkg/operator/k8s_helpers.go

```diff
@@ -0,0 +1,176 @@
+package operator
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/eclipse/che-operator/pkg/util"
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+	"github.com/sirupsen/logrus"
+	"io"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+	"log"
+	"strconv"
+)
+
+type k8s struct {
+	clientset kubernetes.Interface
+}
+
+func GetK8SConfig() *k8s {
```

Comment:
as far as I see it returns `K8sClient` but not `K8sConfig`. Please consider renaming it.